### PR TITLE
Adding --merge operations to the cli, & other fixes

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR has these changes

- Added a --target-county option to ./prime cli
- Added the --merge file1,file2,file3 option to ./prime cli
- Added tests of merging and of STRAC to quick-test

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [X] Updated the docs?
- [X] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 

- #165

### CODE REVIEW

File: prime-router/docs/releases/2021-01-12-release-notes.md [hunk 1]
```diff
@@ -15,4 +15,4 @@ Added this value to `Specimen_type_code`
 
 ## Changes for AZ
 
-One additional facility is now sending to `TEST/IN`.
+One additional facility is now sending to `PROD/IN`.
```
Comment (jimduff-usds) [inline prime-router/docs/releases/2021-01-12-release-notes.md:18]: Typo.

File: prime-router/quick-test.sh [hunk 2]
```diff
@@ -25,6 +25,7 @@ RUN_FL=0
 # always should run, but we'll leave this here for now in case that could change at some point
 RUN_STANDARD=1
 RUN_ALL=0
+RUN_MERGE=0
```
Comment (jimduff-usds) [inline prime-router/quick-test.sh:29]: Added some tests of merging, and of STRAC schema.

File: prime-router/src/main/kotlin/FakeReport.kt [hunk 3]
```diff
@@ -6,7 +6,11 @@ import java.util.Random
 import java.util.concurrent.TimeUnit
 
 class FakeReport(val metadata: Metadata) {
-    class RowContext(findLookupTable: (String) -> LookupTable? = { null }, reportState: String? = null) {
+    class RowContext(
+        findLookupTable: (String) -> LookupTable? = { null },
+        reportState: String? = null,
+        reportCounty: String? = null
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/FakeReport.kt:12]: Added ability to set the county from the command line.

File: prime-router/src/main/kotlin/cli/main.kt [hunk 4]
```diff
@@ -38,8 +37,9 @@ class RouterCli : CliktCommand(
     printHelpOnEmptyArgs = true,
 ) {
     private val inputSource: InputSource? by mutuallyExclusiveOptions(
+        option("--merge", help = "list of comma-separated files to merge").convert { InputSource.ListOfFilesSource(it) },
         option("--input", help = "<file1>").convert { InputSource.FileSource(it) },
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/cli/main.kt:41]: New --merge takes a list of files, comma separated.   This seemed easier than changing --input to take multiple files, but that would be the more general approach.

**Review by RickHawesUSDS — APPROVED** [review]
Looks good. Could you change the title of the PR from fixes to something like "CLI Merge Operations + fixes"

**Pull request merged**